### PR TITLE
Update to .NET SDK 2.1.400 on Travis, xUnit 2.4, Test SDK 15.8

### PR DESCRIPTION
**Pull request by ctaggart**
Hang tight, just troubleshooting build failure for #12. I think it may be an old version of dotnet sdk on TravisCI.

### CODE REVIEW

**Comment by qmfrederik** [comment]
2.1.400 is not there because Travis stil uses Trusty by default and I think .NET Core dropped support for Trusty.

You should be able to use dist: xenial in the travis-ci.yml file, though, to have the builds run on Xenial. There's not a lot of doc but https://github.com/travis-ci/travis-yml/issues/15 has some background.

Alternatively, it's fine with me if the SourceLink indexing only runs on AppVeyor for now - we can publish the packages from there.

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ before_script:
 - kubectl cluster-info
 
 script:
+- dotnet --info
```
Comment (ctaggart) [inline .travis.yml:15]: Happy to comment this out if you wish.

File: appveyor.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 install:
+  - dotnet --info
```
Comment (ctaggart) [inline appveyor.yml:2]: Let me know if you want this commented out or removed.

**Review by qmfrederik — APPROVED** [review]

File: ci/install.sh [hunk 3]
```diff
@@ -1,9 +1,12 @@
+#!/bin/bash -eux
```
Comment (ctaggart) [inline ci/install.sh:1]: I'd leave in the `e` & `u` for error handling. Let me know if you want the `x` removed which just echos the commands. Seems pretty useful.

**Comment by qmfrederik** [comment]
Awesome, thanks!

**Comment by ctaggart** [comment]
Assuming this works, can you **squash** merge it with the squash merge button?

**Comment by ctaggart** [comment]
Looks like it hung starting the tests. I'll check after lunch.

**Comment by qmfrederik** [comment]
@ctaggart Squashed & merged, thank you for doing this!

**Comment by ctaggart** [comment]
Thanks for building the library. I'm trying to use it and running into an exception, which is why I want source link enabled.

**Pull request merged**